src/App.jsx

```diff
@@ -14,33 +15,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_ALPHABET = 'alphabet';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getVisibleGoods(goods, { sortField, reverse }) {
+  const visibleGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    visibleGoods.sort((g1, g2) => {
+      switch (sortField) {
+        case SORT_FIELD_LENGTH:
+          return g1[sortField] - g2[sortField];
+        case SORT_FIELD_ALPHABET:
+          return g1.localeCompare(g2);
+        default:
+          return 0;
+      }
+    });
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (reverse) {
```

Comment:
[Not critical]
We can handle reverse order inside of sorting function, something like:
```
case SORT_FIELD_LENGTH:
    return reverse ? g2[sortField] - g1[sortField] : g1[sortField] - g2[sortField];
```
This would help to achieve the same result with less iterations over the array.